### PR TITLE
Mocute Controller Support

### PR DESCRIPTION
**Pull request by edgarjcfn**
Mocute controllers follow the same principle as iCade, but have different mapping.

### CODE REVIEW

**Comment by ivanaxel** [comment]
b

**Comment by sevdestruct** [comment]
product-image-253795775
61xfpfq3ril _sl1500_

**Comment by QuarterSwede** [comment]
The Mocute Gamepad should work in iCade mode. Turn iCade mode on in Provenance settings and connect.

**Comment by sevdestruct** [comment]
@QuarterSwede: should? can you confirm? Or someone else who has this controller? I don't have one of these.

**Comment by QuarterSwede** [comment]
I cannot confirm. I just looked st the manual for it and it uses iCade mode for iOS.

**Comment by jasarien** [comment]
Well, if you read the original comment for this PR, it says exactly that — “it uses the same principal as iCade controllers, but uses a different mapping”. Therefore this PR provides that different mapping.

**Comment by QuarterSwede** [comment]
Apologies. Must have missed that on the emailed version.

File: ProvenanceTV/Info.plist [hunk 1]
```diff
@@ -32,7 +32,7 @@
 		</dict>
 	</array>
 	<key>CFBundleVersion</key>
-	<string>651</string>
+	<string>683</string>
```
Comment (jasarien) [inline ProvenanceTV/Info.plist]: Thsee kinds of changes should be removed from the PR.

File: ProvenanceTV/Info.plist [hunk 2]
```diff
@@ -41,7 +41,7 @@
 		<true/>
 	</dict>
 	<key>Revision</key>
-	<string>7c309de</string>
+	<string>439486d</string>
```
Comment (jasarien) [inline ProvenanceTV/Info.plist]: These kinds of changes should be removed from the PR.

File: Provenance/Provenance-Info.plist [hunk 3]
```diff
@@ -38,7 +38,7 @@
 	<key>CFBundleSignature</key>
 	<string>????</string>
 	<key>CFBundleVersion</key>
-	<string>678</string>
```
Comment (jasarien) [inline Provenance/Provenance-Info.plist]: These kinds of changes should be removed from the PR.

File: Provenance/Provenance-Info.plist [hunk 4]
```diff
@@ -53,7 +53,7 @@
 	<key>NSPhotoLibraryUsageDescription</key>
 	<string>Provenance can set custom artworks from your photos or save screenshots to your photos library.</string>
 	<key>Revision</key>
-	<string>59b50b9</string>
```
Comment (jasarien) [inline Provenance/Provenance-Info.plist]: These kinds of changes should be removed from the PR.

**Comment by sevdestruct** [comment]
Thanks @jasarien. To that point I updated this issue with a pic, because I didn't know what controller this was, and I want someone with one of these to review the PR with a proper test and confirm this works and is without conflict so we can merge and close the ticket or provide feedback, not just taking a guess based on the documentation that it should work. 

----
So... can someone who has this controller review and confirm this fix? 
----

Additionally, if the pic I posted isn't the exact mocute… the model wasn't specified by @edgarjcfn, so I wanted to find out if this works for all Mocutes models or just one specifically? Thank you.

**Comment by Cavallio** [comment]
Greetings, I’m sorry I’m new to this. (This seems the most relevant post I could find)
I have just bought a Mocute -050 controller. In its settings it says icade however the buttons do not function properly when in use with provenance. Is a fix been worked on?

**Comment by JoeMatt** [comment]
@Cavallio There's iCade and Mocute mode in 1.5 builds. Please direct questions to the Discord server, links found in the readme and main page.

**Pull request closed without merging**